### PR TITLE
Changes for v0.27.1.1

### PR DESCRIPTION
**Pull request by clanmills**
This is an WIP "umbrella".   This will include fixes for #743, #744 and #745.  Hope to be ready for review later this week.  I would like to delay v0.27.1.1 (v0.27.1 RC1) until exiv2.org is moved and stable.

Thanks to @piponazo for the following instructions:
```
git checkout 0.27-maintenance # Move to that branch
git pull --rebase                            # Bring latest changes from the remote
git checkout -b 0.27.1-XXX        # Create a new branch 
#Do some work there (commits)
git push --set-upstream origin 0.27.1-XXX # push the branch to the server
```

And thanks to @D4N and @cryptomilk for help with git.  One day I might understand!

### CODE REVIEW

**Comment by D4N** [comment]
Robin, please drop the commits b40f5f0 and 1a564a4, these two commits are already in 0.27-maintenance (as 7bd929de31622a07bfc16f1c61b6c45e79e69cbb and 5e6c2855ce98bd5fa89e84a51049cd3b90a5c140). Your branch is stale, that's why they aren't there. I can update it for you if you want to.

**Comment by clanmills** [comment]
Dan.  Can you update it please.  I couldn't understand how those we there.  One day I might understand git.  Thanks.

Working my way through all our documentation and stuff and expect to have v0.27.1 RC1 on the new web-site this evening.

**Comment by D4N** [comment]
> Dan. Can you update it please.

Done

**Comment by clanmills** [comment]
Thanks Dan for fixing the git nonesense in dd15ae9 and 420aed8

Exiv2 v0.27.1 RC1 is available for review/inspection:  https://pre-release.exiv2.org/index.html

A huge thank you to everybody who has contributed.  You are mentioned in the release notes.  Thanks to @nehaljwani and @ahuggel for cooperating to move exiv2.org to a new host.

There are open issues can either be fixed, or deferred:  https://github.com/Exiv2/exiv2/milestone/3

I believe the "MacOS-X cruft" issue #620 has been fixed.  I'd like to ask @cryptomilk and @a17r to inspect the release and close the issue.

I hoped to reach Exiv2 v0.27.1 GM this weekend and delayed by the website transfer.  If we elect to make code further code changes, I'll issue RC2 when they have been made.  If we elect to promote RC1 to GM, there will be minor documentation updates. 

If nobody says "STOP" I will announce this on Facebook and Pixls on Sunday 31 March which was the schedule date of Exiv2 v0.27.1 GM.

File: README.md [hunk 1]
```diff
@@ -45,9 +45,9 @@ write, delete and modify Exif, IPTC, XMP and ICC image metadata.
 | Exiv2 Resource              | Location |
 |:------                      |:----     |
 | Project Homepage            | [https://github.com/Exiv2/exiv2](https://github.com/Exiv2/exiv2) |
-| Downloads and Documentation | [http://exiv2.dyndns.org](http://exiv2.dyndns.org:8080) |
-| BuildServer:                | [http://exiv2.dyndns.org:8080](http://exiv2.dyndns.org:8080) |
-| License (GPLv2)             | [license.txt](license.txt) |
+| Downloads and Documentation | [https://exiv2.org](https://exiv2.org) |
+| Prereleases:                | [https://prerelease.exiv2.org](https://prerelease.exiv2.org) |
```
Comment (nehaljwani) [inline README.md]:
```suggestion
| Prereleases:                | [https://pre-release.exiv2.org](https://pre-release.exiv2.org) |
```

Comment (clanmills) [reply README.md]: Sure.  That's a typo and I'll fix it on the next "round".  There will be more typos because I'm dyslexic.

**Review by piponazo — APPROVED** [review]

**Comment by piponazo** [comment]
Once the CI is green I will merge and create a new tag as requested in #766

**Comment by a17r** [comment]
lgtm!

**Comment by clanmills** [comment]
Gentlemen:

Nobody has made any negative comments about the code in v0.27.1.RC1.  If the code v0.27.1.RC1 => v0.27.0, we don't need this tag.

There are a few open issues on Exiv2 v0.27.1
https://github.com/Exiv2/exiv2/milestone/3

I'd like to defer those issues for v0.27.2 and change nothing else.

So 0.27.1 = the code in "Changes_for_v0.27.1.1" plus a few documentation changes.  And of course, the CMake/Project() information in CMakeLists.txt.

**Comment by D4N** [comment]
The tag has been out only for an hour or so, I'd give downstream a
little more time, they probably didn't even get a chance to test this
yet.

**Comment by clanmills** [comment]
Quite so, Dan.

The point I'm making is that we don't need a tag if the code v0.27.1 == code v0.27.1-RC1, we don't need an additional tag.  However let's wait another week as something may arise.  

With v0.27.0 RC1, RC2, RC3 and GM, I received feedback almost within hours of the announcement.

**Pull request merged**